### PR TITLE
Adding geemap package

### PR DESCRIPTION
**Pull request by giswqs**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by giswqs** [comment]
@conda-forge/help-python It is ready for review. Thanks.

File: recipes/geemap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 311b4c935575b41819c30d30ac01b7788e1f1d61e996734fb5f4a44945c6b32a
+
+build:
+  number: 0
```
Comment (npavlovikj) [inline recipes/geemap/meta.yaml]: I believe the recipe can be `noarch: python` ?

File: recipes/geemap/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  entry_points:
+    - geemap=geemap.cli:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```
Comment (npavlovikj) [inline recipes/geemap/meta.yaml]: Usually `pip` and `python` are enough for host dependencies.

File: recipes/geemap/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  imports:
+    - geemap
+  commands:
+    - geemap --help
+
+about:
+  home: "https://github.com/giswqs/geemap"
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (npavlovikj) [inline recipes/geemap/meta.yaml]: Please add license file. If it is not part of the tarball, you can copy it from the repo.

**Comment by giswqs** [comment]
@npavlovikj I have made changes as suggested. Thank you.

**Comment by npavlovikj** [comment]
Thank you for contributing and the prompt fixes @giswqs !

**Pull request merged**